### PR TITLE
Configurable transaction statistics and dynamic fees - Closes #291

### LINKED ISSUE #291: Make switch for background operations in config
### Description

Make it possible to disable the following background operations:

- Fee estimator (quick calculation)
- Fee estimator (full calculation)
- Transaction statistics

### Motivation

- It saves time during testing and debugging

### Acceptance Criteria

The described options are available in the config with proper environment variables.

### PR DESCRIPTION
**Pull request by MichalTuleja**
### What was the problem?
This PR resolves #291

### How was it solved?

- [x] Fee estimator (quick calculation)
- [x] Fee estimator (full calculation)
- [x] Transaction statistics

### How was it tested?

Jenkins

### CODE REVIEW

File: services/core/jobs/transactionStatistics.js [hunk 1]
```diff
@@ -26,12 +26,16 @@ module.exports = [
 		schedule: '*/30 * * * *', // Every 30 min
 		updateOnInit: true,
 		init: () => {
-			logger.debug('Scheduling delegate list init...');
-			transactionStatistics.init(config.transactionStatistics.historyLengthDays);
+			if (config.transactionStatistics.enabled) {
+				logger.debug('Scheduling delegate list init...');
```
Comment (sameersubudhi) [inline services/core/jobs/transactionStatistics.js:30]:
```suggestion
				logger.debug('Initiating transaction statistics computation.');
```

File: services/core/jobs/transactionStatistics.js [hunk 2]
```diff
@@ -26,12 +26,16 @@ module.exports = [
...
+				transactionStatistics.init(config.transactionStatistics.historyLengthDays);
+			}
 		},
 		controller: async () => {
-			logger.debug('Scheduling delegate list reload...');
-			transactionStatistics.updateTodayStats();
+			if (config.transactionStatistics.enabled) {
+				logger.debug('Scheduling delegate list reload...');
```
Comment (sameersubudhi) [inline services/core/jobs/transactionStatistics.js:36]:
```suggestion
				logger.debug('Job scheduled to update transaction statistics.');
```

**Review by sameersubudhi — APPROVED** [review]
Everything looks good except the debug logs in `services/core/jobs/transactionStatistics.js`.

**Review by nagdahimanshu — APPROVED** [review]
Looks good!

**Pull request merged**